### ISSUE TITLE
examples don't work

### ISSUE THREAD

**Issue by kofifus** [chafey/cornerstoneWADOImageLoader#75]
None of the examples in https://rawgit.com/chafey/cornerstoneWADOImageLoader/master/examples/index.html work with the given links 

(except for the "dicomfile (DICOM P10 on local file system)")

**Comment by swederik** [comment]
It looks like the links are pointing to the dev branch which I had deleted yesterday... 

Thanks for the report!

**Comment by swederik** [comment]
I've fixed most of the examples.

The WADO-RS example is throwing an error which appears to be related to this commit: https://github.com/chafey/cornerstoneWADOImageLoader/commit/29497d2a14fcbee73b1d3df6419cac9d78d27992

**Comment by kofifus** [comment]
thx!

As for the multiframe example https://rawgit.com/chafey/cornerstoneWADOImageLoader/master/examples/wadourimultiframe/index.html

how do I change frames ? or get them to play ?

**Comment by swederik** [comment]
Hmm good question. It looks like it should be reading the number of frames from the DICOM and playing them as a clip in a stack:

https://github.com/chafey/cornerstoneWADOImageLoader/blob/master/examples/wadourimultiframe/index.html#L145

If it's not doing that, then I guess something is wrong with it. Thanks for the report!

**Comment by rdsmartins** [comment]
I tried the example at : https://rawgit.com/chafey/cornerstoneWADOImageLoader/master/examples/wadors/index.html and it also didn't work.

I tried to api link: https://api.hackathon.siim.org/dicomweb/studies/1.3.6.1.4.1.14519.5.2.1.7777.9002.198875685720513246512710453733/series/1.3.6.1.4.1.14519.5.2.1.7777.9002.207203214132667549392101803048/instances/1.3.6.1.4.1.14519.5.2.1.7777.9002.327873213718058651550666129029/frames/1

But it seems it requires an API Key. 

And the suggested URL: ../../testImages/wadors fails ...

Does anyone have an idea ?

Tks.
